### DISCUSSION TITLE
Running costs?

### DISCUSSION THREAD

**Discussion by andrewpatto** [umccr/elsa-data#46] [General]
A slightly left field question - but what is a running cost for Elsa Data that would be problematic?

So the cloud deployment of Elsa Data will obviously have some AWS (or Google) running costs - but part of the architectural design allows you to trade costs for each of development (say).. or trades redundancy for costs.

it is therefore generally useful for the software architects to have an envelope "budget" for running costs to help shape that decision making.

So what is monthly budget for Elsa Data that would make you go "hmm.. that's going to be a stretch". i.e the amount that given how you currently understand its functionality/role - would make you feel you would need to have a hard discussion with management in order to get it allowed into your org.

We can keep it to a $10, $100, $1000 / month.. level of discussion..

I will start things off - I think it would be hard to justify spending $100k/month on running Elsa Data for any organisation - even those with a lot of data/datasets.

I think any organisation quibbling over spending $1 or $10/month on running a piece of software is probably not serious.

So we are between $10 - $100k per month.. over to you..

@sarahcasauria @ohofmann

**Comment by andrewpatto** [comment]
To give some idea - I think the running costs of the current stack in AWS would be $50-$100/month (though that is missing some bits, but also with no cost optimisation done). So starting from the premise that running it for under $50/month would likely be difficult

**Comment by ohofmann** [comment]
Our current cost for AWS is around 10k a month. It would be nice if we could keep Elsa to 10% of that - beyond it I need to start looking into long term funding sources.
